package.json

```diff
@@ -25,6 +25,7 @@
     "packages/util",
     "packages/openapi-generator",
     "packages/generator-common",
+    "packages/temporal-middleware",
```

Comment:
I am thinking to rename this to `temporal-de-serializers`, because `middleware` might be a less known keyword in our context. What do you think?

Reply:
Since in our code base we also have the `deSerializer` name all around I would also stick to this and also have a look at our documentation.